recipes/doconce/meta.yaml

```diff
@@ -0,0 +1,99 @@
+{% set name = "doconce" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "276f7ad7fcc5467c3329b9260624c6a12a6b5a30be265132de52e9d17a6da7eb" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/hplgit/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install
+  skip: True  # [py3k]
+
+requirements:
+  build:
+    - python
+    - pygments
```

Comment:
I know `doconce` doesn't use `setuptools` currently. However, if it does in the future, note that `pygments` will pull it in as a run time dependency. Just something to keep in mind going forward.